zwave/config.json

```diff
@@ -25,11 +25,13 @@
   "devices": ["/dev/bus/usb:/dev/bus/usb:rwm", "/dev/mem:/dev/mem:rw"],
   "options": {
     "device": null,
-    "network_key": ""
+    "network_key": "",
+    "instance": 1
```

Comment:
Let's remove this from the default configuration. Most people should not need this and can be confused by this setting.

We can still allow setting it, document it, and check on runtime if it is set.

Reply:
OK, I think I've properly changed this.